### ISSUE TITLE
On MacOS Big Sur FS can't fimd pthreads library

### ISSUE THREAD

**Issue by pathron** [FlexibleSUSY/FlexibleSUSY#292]
This was reported here https://github.com/GambitBSM/gambit/issues/254 for an old version of FS that is in GAMBIT, but  looking at the latest configure i think it should apply to the latest development branch too.  I asked Jonathan to confirm this if he can. 

As per that thread it seems MacOS have renamed and moved a lot of libraries around and this is creating these kind of problems.

Looks to me like we can fix it by adding  `/usr/lib/system`as a path to check and also looking for libraries with the name `libsystem_pthread`. If no-one can test it on Big Sur I may be able to ask Jonathan to test it for us.

I wonder if this MacOS was named after this https://en.wikipedia.org/wiki/Big_Sur_(novel) rather than just the place in California.  One of my favorite novels, but it does basically describe Kerouac descending into  paranoia and having a nervous breakdown...

**Comment by pathron** [comment]
I just tagged everyone who comes up in case one of you have MacOS BigSur or an older one and want to update to it.

**Comment by pathron** [comment]
Sorry if that caused spam I was trying to add @uukhas to the assignees, but its not working...

**Comment by wkotlarski** [comment]
@pathron, I can't open [this](https://github.com/GambitBSM/gambit/issues/254) link.

**Comment by pathron** [comment]
Yeah sorry I realised a bit later on when discussing with Alex that you can't see the link as the isue page is for the private gambit repo not the public forks of release branches . There were some annoying complications and we have been discussing a while off thread and think we have a fix, but I am waiting on my GAMBIT collaborator who has the BigSur to test some commands.  I also have an uglier solution if these don't work. 
  
If you happen to have access to a mac with BigSur let me know, but otherwise don't worry about it.  I may also try Dylan since he used to have a mac at least.

**Comment by pathron** [comment]
I will open a new issue for the related fact that we don't actually disable multi-threading if finding pthreads fails.
